boskos/ranch/ranch.go

```diff
@@ -98,6 +100,11 @@ func NewRanch(config string, s *Storage) (*Ranch, error) {
 	return newRanch, nil
 }
 
+// typeState is used as keu
+type typeState struct {
```

Comment:
could use better godoc and longer field names

Reply:
done